teamscale-gradle-plugin/build.gradle

```diff
@@ -17,48 +16,32 @@ gradlePlugin {
 
 version pluginVersion
 
-configurations {
-	testImplementation.extendsFrom(shadow)
-}
-
 dependencies {
-	shadow gradleApi()
-	shadow 'org.jetbrains.kotlin:kotlin-stdlib:1.3.11'
-
 	implementation project(':teamscale-client')
 	implementation project(':report-generator')
+
+	implementation gradleApi()
+	implementation 'org.jetbrains.kotlin:kotlin-stdlib:1.3.11'
+
 	implementation 'org.eclipse.jgit:org.eclipse.jgit:5.0.2.201807311906-r'
 	implementation group: 'com.squareup.retrofit2', name: 'converter-gson', version: '2.4.0'
 
 	testImplementation 'junit:junit:4.12'
 	testImplementation 'org.assertj:assertj-core:3.8.0'
 	testImplementation 'org.junit.jupiter:junit-jupiter-api:5.4.0'
+
 	testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.4.0'
 	testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.4.0'
 }
 
-// Is also added to compile by java-gradle-plugin.
-// We manually add it to shadow and remove it from compile.
-configurations.compile.dependencies.remove dependencies.gradleApi()
-
 test {
 	testLogging.exceptionFormat "full"
 }
 
-// Needed to bundle org.conqat.lib.commons into the report-generator
-// as this dependency is not available via mavenCentral
-shadowJar {
-	classifier = null
-	manifest {
-		attributes 'Implementation-Version': pluginVersion
-		attributes 'Specification-Version': appVersion
```

Comment:
We need to keep this manifest stuff for the normal jar file or only use the constants in BuildVersions.kt

Maybe switching to the normal (non-shaded) jar gets us rid of having to specify the version by hand. Need's to be check tough if they are available from the test.

Reply:
The tests for the Gradle plugin still only work locally due to the unreleased new version of the agent included in this PR. Is this sufficient or do we need to check more?

Reply:
I think you misunderstood me ;-)
Currently we use these manifest entries to determine at runtime which version to retrieve for the agent and the impacted-test-engine. For the test environment we did not have these as they were only part of the shaded plugin. That's why we additionally needed to specify these in BuildVersion.kt as fallback.

If we specify the manifest entries for the jar task like:
```
jar {
	manifest {
		attributes 'Implementation-Version': pluginVersion
		attributes 'Specification-Version': appVersion
	}
}
```
We might be able to retrieve them also during the test scenario and therefore no longer need to specify these manually in BuildVersions.kt